src/command_modules/azure-cli-storage/azure/cli/command_modules/storage/services_wrapper.py

```diff
@@ -41,7 +42,15 @@ def add_cors(self, origins, methods, max_age, exposed_headers=None, allowed_head
         cors = self.get_cors(timeout)
         new_rule = CorsRule(origins, methods, max_age, exposed_headers, allowed_headers)
         cors.append(new_rule)
-        return self.set_service_properties()(cors=cors, timeout=timeout)
+        try:
+            return self.set_service_properties()(cors=cors, timeout=timeout)
+        except AzureHttpError as ex:
+            import re
+            message = re.search(r"([A-Za-z\t .])+", str(ex))
+            if message.group(0) == 'XML specified is not syntactically valid.':
```

Comment:
Is this a service workaround?  If so, we should create a tracking issue so we can remove this when (!) the service fixes its ways.